INSTRUMENTS/SERIAL/SERIAL_TIMESERIES/SERIAL_TIMESERIES.py

```diff
@@ -46,10 +33,10 @@ def SERIAL_TIMESERIES(dc_inputs, params):
     num_readings * record_period is roughly the run length in seconds.
     """
 
-    COM_PORT = params["comport"]
-    BAUD = params["baudrate"]
-    NUM = params["num_readings"]
-    RECORD_PERIOD = params["record_period"]
+    COM_PORT = comport
```

Comment:
It's unnecessary to redefine these variables. we can directly use them across the function body.